website/docs/index.md

```diff
@@ -14,7 +14,7 @@ hero:
       text: Explore Github
       link: https://github.com/semrush/intergalactic
   image:
-    src: /halloween.png
```

Comment:
maybe we should remove halloween.png file too?

Reply:
I saw that we still have other holiday illustrations in the same folder (at least one), so I thought it might be convenient to save them until next occasion. But if there's a reason to remove them, I can do that